modules/462-loki/templates/statefulset.yaml

```diff
@@ -58,6 +62,13 @@ spec:
         {{- include "helm_lib_module_container_security_context_read_only_root_filesystem" . | nindent 8 }}
         image: {{ include "helm_lib_module_image" (list . "loki") }}
         command: [ '/usr/bin/loki', '-config.file=/etc/loki/config.yaml' ]
+        env:
+          - name: PROMETHEUS_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: prometheus-token
+                key: token
+                optional: false
```

Comment:
For Loki, prometheus token is not required if I get it right.
```suggestion
```

Reply:
done